### DISCUSSION TITLE
`.env.template` mentions nonexistent wiki

### DISCUSSION THREAD

**Discussion by 57194** [dani-garcia/vaultwarden#4836] [Q&A]
`.env.template` mentions a wiki that I don't think exists? I can't find one, anyway. Should either remove the note saying to check the wiki or create the page and link to it.

https://github.com/dani-garcia/vaultwarden/blob/a1204cc9350366a06d9a2dc8554aedf43aa6f442/.env.template#L332-L336

---

Edit: Should also probably state if what's there in the comments is actually true and how this interacts with or overrides `HTTP_REQUEST_BLOCK_NON_GLOBAL_IPS`.

https://github.com/dani-garcia/vaultwarden/blob/a1204cc9350366a06d9a2dc8554aedf43aa6f442/.env.template#L338-L340

**Comment by alexpovel** [comment]
There is a wiki here: https://github.com/dani-garcia/vaultwarden/wiki

**Reply by 57194** [reply]
The issue is that it _just_ says "check the wiki", but _what page_ on the wiki is actually relevant to this configuration option?
